### PR TITLE
add created_at to visualization properties

### LINKED ISSUE #9191: [Metrics] Add 'age' attribute to deletion-related events
Add new attribute 'age' (calculated as 'deletion time' - 'creation time', in days) for following events:
 - Deleted dataset
 - Deleted map

This shouldn't be difficult as both maps and datasets have a created_at property.

### PR DESCRIPTION
**Pull request by gfiorav**
Fixes #9191 
Fixes https://github.com/CartoDB/cartodb/issues/9271

### CODE REVIEW

**Comment by gfiorav** [comment]
Please CR, @javitonino

File: lib/carto/tracking/events.rb [hunk 1]
```diff
@@ -3,11 +3,15 @@
...
+
         properties = {
-          vis_id: table_visualization.id,
-          privacy: table_visualization.privacy,
-          type: table_visualization.type
+          vis_id: visualization.id,
+          privacy: visualization.privacy,
+          type: visualization.type,
+          created_at: created_at,
+          lifetime: Time.now.utc - created_at
```
Comment (javitonino) [inline lib/carto/tracking/events.rb]: This returns seconds, but days were requested in the ticket. Don't know if it is important (maybe seconds are better, more granularity)

File: lib/carto/tracking/events.rb [hunk 2]
```diff
@@ -3,11 +3,15 @@
...
+        created_at = visualization.created_at
+
         properties = {
-          vis_id: table_visualization.id,
-          privacy: table_visualization.privacy,
-          type: table_visualization.type
+          vis_id: visualization.id,
+          privacy: visualization.privacy,
+          type: visualization.type,
+          created_at: created_at,
```
Comment (javitonino) [inline lib/carto/tracking/events.rb]: Check that this date format is parseable by Segment. I guess it is, but I don't know how this works exactly.

**Comment by javitonino** [comment]
Take into account the comments you want, and ship it!

File: lib/carto/tracking/events.rb (hunk 2 above)
Comment (gfiorav) [inline lib/carto/tracking/events.rb]: yup, we've been using it already! https://github.com/CartoDB/cartodb/blob/master/lib/carto/tracking/events.rb#L56

Comment (javitonino) [inline lib/carto/tracking/events.rb]: Awesome :sparkles:

File: lib/carto/tracking/events.rb (hunk 1 above)
Comment (gfiorav) [inline lib/carto/tracking/events.rb]: True! I asked in the original ticket in case we'd want' to reevaluate granularity. Sublime catch!

**Comment by gfiorav** [comment]
I changed the units to days (with decimal) and added some properties to the event general properties. Review again please @javitonino and let's put this in prod as soon as posible!

File: lib/carto/tracking/events.rb [hunk 3]
```diff
@@ -3,11 +3,16 @@
 module Carto
   module Tracking
     module PropertiesHelper
-      def visualization_properties(table_visualization, origin: nil)
+      def visualization_properties(visualization, origin: nil)
+        created_at = visualization.created_at
+        lifetime_in_days_with_decimal = (Time.now.utc - created_at).to_f / 60 / 60 / 24
```
Comment (javitonino) [inline lib/carto/tracking/events.rb]: Ruby not having proper timediff objects is a bit disappointing. :cry: 

Anyway, can you extract this to a function? It is repeated in `event_properties`.

**Comment by javitonino** [comment]
Other than the proposed refactor (extract timediff function), nothing else to add. Ship when :green_apple:

**Comment by javitonino** [comment]
:shipit:

**Pull request merged**